joomla/joomla-cms
# [5.0] Added missing "items" text to the Joomla update view

Pull Request for Issue #39829.

### Summary of Changes
Added missing "items" text to the Joomla update view.


### Testing Instructions
After the patch is applied you have to update your Joomla without reverting the code change of the patch!
Maybe there is an easier/better approach to do but you could follow these steps:

1. Apply the patch
2. Create a .zip file from your components folder and store that .zip file outside of the Joomla folder. It is used later to regularly update Joomla (which will just copy the same files).
3. Login into the backend (administrator area).
4. In the left hand toggle menu click "System".
5. In the "Update" section click "Joomla".
6. Click the button "Upload and Update".
7. Choose your .zip file (from step 2).
8. Check the box "I'm aware that a backup before any update is highly recommended. ".
9. Click the button "Upload & Install".
10. Confirm your credentials.
11. Click button "Install". 


### Actual result BEFORE applying this Pull Request
In the third column only the number of extracted/processed items is visualized:

![Screenshot 2023-06-08 192950](https://github.com/joomla/joomla-cms/assets/8353850/ab599fdb-0600-41ab-b9f2-18580a4882e5)



### Expected result AFTER applying this Pull Request
In the third column the number of extracted/processed items is visualized with an additional text "items":

![Screenshot 2023-06-10 145523](https://github.com/joomla/joomla-cms/assets/8353850/2ac74ab0-7788-404c-9d55-c924d0d0908c)





### Link to documentations
Please select:
